service/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateAzureLandingZoneStepTest.java

```diff
@@ -0,0 +1,86 @@
+package bio.terra.landingzone.stairway.flight.create;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.landingzone.library.LandingZoneManagerProvider;
+import bio.terra.landingzone.library.landingzones.management.LandingZoneManager;
+import bio.terra.landingzone.library.landingzones.management.deleterules.LandingZoneRuleDeleteException;
+import bio.terra.landingzone.model.LandingZoneTarget;
+import bio.terra.landingzone.stairway.flight.LandingZoneFlightMapKeys;
+import bio.terra.profile.model.ProfileModel;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.StepStatus;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@Tag("unit")
+class CreateAzureLandingZoneStepTest {
+
+  private static final UUID LANDING_ZONE_ID = UUID.randomUUID();
+  private static final UUID TENANT_ID = UUID.randomUUID();
+  private static final UUID SUBSCRIPTION_ID = UUID.randomUUID();
+  private static final String MRG_ID = "mymrg";
+  private ProfileModel billingProfile;
+  @Mock private CreateAzureLandingZoneStep createAzureLandingZoneStep;
```

Comment:
If I am not mistaken createAzureLandingZoneStep is SUT. We probably need real instance? It is initialized in setUp. So, probably need to remove @Mock.

Reply:
Done